From 95578e5e2d23e47cf8bd9896936480d121a24290 Mon Sep 17 00:00:00 2001
From: "hjnim.kim" <hjnim.kim@samsung.com>
Date: Thu, 7 Nov 2013 19:15:19 +0900
Subject: [PATCH] JQM popup bg close event handler is only works when popup is
 opend

JQM Popup bg close event handler is only works in popup is opend ( this._isOpen = true )

Signed-off-by : hyunjung kim <hjnim.kim@samsung.com>

Change-Id: I637c5336f075f8be4acf89b4738bf4b5dc558d99
---
 libs/js/jquery-mobile-1.2.0/js/widgets/popup.js |    8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/libs/js/jquery-mobile-1.2.0/js/widgets/popup.js b/libs/js/jquery-mobile-1.2.0/js/widgets/popup.js
index 59eac80..7661836 100644
--- a/libs/js/jquery-mobile-1.2.0/js/widgets/popup.js
+++ b/libs/js/jquery-mobile-1.2.0/js/widgets/popup.js
@@ -62,9 +62,11 @@ define( [ "jquery",
 		},
 
 		_eatEventAndClose: function( e ) {
-			e.preventDefault();
-			e.stopImmediatePropagation();
-			this.close();
+			if( this._isOpen ) {
+				e.preventDefault();
+				e.stopImmediatePropagation();
+				this.close();
+			}
 			return false;
 		},
 
-- 
1.7.9.5

